src/lib/sitemap/tutorials-content-urls.ts

```diff
@@ -0,0 +1,55 @@
+import { getAllCollections } from 'lib/learn-client/api/collection'
+import { SectionOption } from 'lib/learn-client/types'
+import { activeProductSlugs } from 'lib/products'
+import { generateTutorialMap } from 'pages/api/tutorials-map'
+import { ProductSlug } from 'types/products'
+import { getCollectionSlug } from 'views/collection-view/helpers'
+import { makeSitemapElement } from './helpers'
+import { Collection as ClientCollection } from 'lib/learn-client/types'
+
+function getTutorialLandingPaths(): string[] {
+	const activeSlugs = activeProductSlugs.map(
+		(productSlug: ProductSlug) => `/${productSlug}/tutorials`
+	)
+	const sectionOptionsWithLandingPage = Object.values(SectionOption).filter(
+		(option: SectionOption) => option === SectionOption.onboarding
+	)
+	const sectionOptions = sectionOptionsWithLandingPage.map(
+		(slug: SectionOption) => `/${slug}`
+	)
+
+	return [...activeSlugs, ...sectionOptions]
+}
+
+async function getCollectionAndTutorialPaths() {
+	const collectionPaths = []
+	const tutorialPaths = []
+	const allCollections = await getAllCollections()
+
+	allCollections.forEach((collection: ClientCollection) => {
+		// build collection paths
+		collectionPaths.push(getCollectionSlug(collection.slug))
+	})
+
+	return [...collectionPaths, ...tutorialPaths]
```

Comment:
```suggestion
	const allCollections = await getAllCollections()

	return allCollections.map((collection: ClientCollection) => getCollectionSlug(collection.slug)))
```
Didn't test this, but just suggesting a version without the tutorial paths.

Reply:
Sounds good, thank you!